chroma_feedback/producer/circle/core.py

```diff
@@ -22,17 +24,23 @@ def run() -> List[Dict[str, Any]]:
 
 	if ARGS.circle_slug:
 		for slug in ARGS.circle_slug:
-			result.extend(fetch(ARGS.circle_host, None, slug, ARGS.circle_token))
+			result.extend(fetch(ARGS.circle_host, None, slug, ARGS.circle_token, ARGS.circle_only_mine))
 	elif ARGS.circle_organization:
-		result.extend(fetch(ARGS.circle_host, ARGS.circle_organization, None, ARGS.circle_token))
+		result.extend(fetch(ARGS.circle_host, ARGS.circle_organization, None, ARGS.circle_token, ARGS.circle_only_mine))
 	return result
 
 
-def fetch(host : str, organization : str, slug : str, token : str) -> List[Dict[str, Any]]:
+def fetch(host : str, organization : str, slug : str, token : str, only_mine : str) -> List[Dict[str, Any]]:
 	result = []
 	response = None
 
-	if host and slug and token:
+	if only_mine and bool(strtobool(only_mine)) and host and slug and token:
```

Comment:
Remove the type casting and move only_mine to the end according to the function parrams